web/src/components/network/index.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) [2022] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+export { default as Network } from "./Network";
```

Comment:
What about renaming this compononent to `NetworkOverview`. Tha way, we can go for a `NetworkOverview` or `NetworkSummary` for the summary and `NetworkPage` for the section page. Simlarly, we could have `StorageOverview` and `StoragePage`, `LanguageOverview` and `LanguagePage` and so on. What is more, they could be simply an `Overview` and `Page` components inside their modules...

Reply:
Yes, I vote for it: `Network/Overview`, `Network/Page`, etc. But maybe in a separate PR.